build_miniforge_osx.sh

```diff
@@ -4,14 +4,15 @@ set -e
 set -x
 
 echo "Installing a fresh version of Miniforge3."
-MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/download/4.8.3-1"
-MINIFORGE_FILE="Miniforge3-4.8.3-1-MacOSX-x86_64.sh"
+MINIFORGE_VERSION="4.10.3-8"
+MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/download/${MINIFORGE_VERSION}"
+MINIFORGE_FILE="Mambaforge-${MINIFORGE_VERSION}-MacOSX-x86_64.sh"
```

Comment:
Should the `Mambaforge` portion also be refactored out?

Reply:
I'm not sure that really helps. The reason I refactored the version number was to make it easier to spot in 2 lines that needed to keep it consistent. Here we already have a variable called `MINIFORGE_FILE`. I'm not sure what else we can do.